src/solver/simulation/include/antares/solver/simulation/solver.hxx

```diff
@@ -167,10 +168,10 @@ public:
             simulation_->prepareClustersInMustRunMode(scratchmap, y);
 
             // 4 - Hydraulic ventilation
-            Benchmarking::Timer timer;
-            hydroManagement.makeVentilation(randomReservoirLevel, state[numSpace], y, scratchmap);
-            timer.stop();
-            pDurationCollector.addDuration("hydro_ventilation", timer.get_duration());
+            durationCollector("hydro_ventilation") << [&]
```

Comment:
All those calls could override timers from other simulations running in the same process, we don't want it